storybook/src/elements/code-panel-template.js

```diff
@@ -0,0 +1,100 @@
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { until } from 'lit-html/directives/until.js';
+import hljs from 'highlight.js';
+
+export const codePanelTemplate = (path) => html`
```

Comment:
Could've been a custom element too I guess

Reply:
What would be the advantages of having this as an element in this case?
Wouldn't it just add unnecessary boilerplate?